docs/DeveloperGuide.adoc

```diff
@@ -780,15 +780,19 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 [appendix]
 == Product Scope
 
+*Target user*: University students
```

Comment:
Recall from tutorial that this target user is rather vague and generic, perhaps narrowing of the scope to a more focused target user would help maintain the workload on the team to produce a more feature complete product at the end of v1.5rc =).

It seems that B4 has the same target audience as this... perhaps the team can look into differentiating from B4? =P